gobblin-core/src/main/java/gobblin/source/extractor/extract/kafka/KafkaExtractor.java

```diff
@@ -41,6 +46,9 @@
 
   protected Iterator<MessageAndOffset> messageIterator;
   protected long nextWatermark;
+  private long totalRecordSize;
```

Comment:
It's better that @ibuenros replaces these with metrics as part of his ticket. There are other places in the kafka adapter that need to add metrics.